src/options.rs

```diff
@@ -37,6 +37,10 @@ pub(crate) struct Options {
   pub(crate) index_sats: bool,
   #[clap(long, short, help = "Use regtest. Equivalent to `--chain regtest`.")]
   pub(crate) regtest: bool,
+  #[clap(long, help = "Authenticate to Bitcoin Core RPC as <RPC_USER>.")]
+  pub(crate) rpc_user: Option<String>,
+  #[clap(long, help = "Authenticate to Bitcoin Core RPC with <RPC_PASS>.")]
+  pub(crate) rpc_pass: Option<String>,
```

Comment:
I notice that both `rpc_user` and `rpc_pass` are optional. What is the behavior when we pass `rpc_user` without `rpc_pass`, and vice versa? We should have test coverage for all four cases. (up/Up/uP/UP)

Reply:
It tries to get it from env or config file. Finally it looks at the cookie file.